index.html

```diff
@@ -1,206 +1,405 @@
-         <button class="btn_3"> html</button>
-         <button class="btn_4">css</button>
-         <button class="btn_5">javascript</button>
-         </div>
-        
-     <button class="btn_2">See Project</button>
-     </div>
-
-     <div class="container">
-        <div> <img src="images/Portfolio_2.png" class="img_2"> </div>
-        <div class="section_1">
-         <h2 class="head_2"> Multi-Post <br> Stories</h2>
-         <ul class="list-ul">
-           <li class="list_1"> CANOPY </li>
-           <li class="list_2"><img src="images/Counter_1.png">Back End Dev</li>
-           <li class="list_2"><img src="images/Counter_2.png">2015</li>
-               </ul>
-         <p>A daily selection of privately 
-           personalized reads; no accounts 
-           or sign-ups required.</p>   
-         <button class="btn_3"> html</button>
-         <button class="btn_4">css</button>
-         <button class="btn_5">javascript</button>
-         </div>
-         <button class="btn_2">See Project</button>
-     </div>
-
-     <div class="container">
-        <div> <img src="images/Portfolio_3.png" class="img_2"> </div>
-        <div class="section_1">
-         <h2> Tonic </h2>
-         <ul class="list-ul">
-           <li class="list_1"> CANOPY </li>
-           <li class="list_2"><img src="images/Counter_1.png">Back End Dev</li>
-           <li class="list_2"><img src="images/Counter_2.png">2015</li>
-               </ul>
-         <p>A daily selection of privately
-            personalized reads; no accounts 
-            or sign-ups required.</p>   
-         <button class="btn_3"> html</button>
-         <button class="btn_4">css</button>
-         <button class="btn_5">javascript</button>
-         </div>
-         <button class="btn_2">See Project</button>
-     </div>
-
-     <div class="container">
-        <div> <img src="images/Portfolio_4.png" class="img_2"> </div>
-        <div class="section_1">
-         <h2 class="head_2"> Multi-Post <br> Stories </h2>
-         <ul class="list-ul">
-           <li class="list_1"> CANOPY </li>
-           <li class="list_2"><img src="images/Counter_1.png">Back End Dev</li>
-           <li class="list_2"><img src="images/Counter_2.png">2015</li>
-               </ul>
-         <p>A daily selection of privately
-            personalized reads; no accounts 
-            or sign-ups required.</p>   
-         <button class="btn_3"> html</button>
-         <button class="btn_4">css</button>
-         <button class="btn_5">javascript</button>
-         </div>
-        <button class="btn_2">See Project</button>
+  </div>
+
+  <!--section-1-->
+
+  <div class="section-2 card mobile-card">
+    <div class="card-img-div">
+      <img src="images/Portfolio_1.png">
+    </div>
+    <div class="card-details-div">
+      <h2> Tonic </h2>
+      <div class="card-highlights">
+        <p>CANOPY</p>
+        <img src="images/Counter_1.png">
+        <p> Back End Dev</p>
```

Comment:
- [ ] Great work up to here, but I think the modification you brought to your code has broken some though, referring to the original design,  that text should be on a single line, see?

<img width="522" alt="1" src="https://user-images.githubusercontent.com/77843050/215269147-0bd92232-d2bb-4fde-99e0-66ae13372df6.png">